scss/tech.scss

```diff
@@ -1,3 +1,56 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: auto;
+  margin: 20px;
+  padding: 40px;
+  position: relative;
+  text-align: center;
+  width: 300px;
+
+  &::before {
+    border: 1px solid;
+    border-color: $earls-green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    right: 15px;
+		position: absolute;
+		bottom: 15px;
+		width: 100%;
+		z-index: -1;
+  }
+}
+
 .tech-logo {
   height: 100px;
+  width: 100px;
+}
+
+.tech-section {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px 0px 150px;
+}
+.tech-title {
+  margin: 10px;
+}
+
+.tech-sub {
+  font-weight: 900;
+  margin: 20px;
```

Comment:
This margin is adding to the one from tech-title and you dont need it in the sides, better to use just margin-bottom